docs/annotations/annotations.md

```diff
@@ -4,25 +4,25 @@ ExternalDNS sources support a number of annotations on the Kubernetes resources
-| Pod          |            | Yes      | Yes               | Yes     |         |                     |   No    |
-| Service      | Yes        | Yes[^1]  | Yes[^1][^2]       | Yes[^3] | Yes     | Yes                 |   No    |
-| Skipper      | Yes        | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |
-| Traefik      |            | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |
+| Source       | controller | hostname | internal-hostname | target  | ttl     | (provider-specific) | ingress | endpoints-type |
+|--------------|------------|----------|-------------------|---------|---------|---------------------|:-------:|:--------------:|
+| Ambassador   |            |          |                   | Yes     | Yes     | Yes                 |   No    |                |
+| Connector    |            |          |                   |         |         |                     |   No    |                |
+| Contour      | Yes        | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |                |
+| CloudFoundry |            |          |                   |         |         |                     |   No    |                |
+| CRD          |            |          |                   |         |         |                     |   No    |                |
+| F5           |            |          |                   | Yes     | Yes     |                     |   No    |                |
+| Gateway      | Yes        | Yes[^1]  |                   | Yes[^4] | Yes     | Yes                 |   No    |                |
+| Gloo         |            |          |                   | Yes     | Yes[^5] | Yes[^5]             |   No    |                |
+| Ingress      | Yes        | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |                |
+| Istio        | Yes        | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   Yes   |                |
+| Kong         |            | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |                |
+| Node         | Yes        |          |                   | Yes     | Yes     |                     |   No    |                |
+| OpenShift    | Yes        | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |                |
+| Pod          |            | Yes      | Yes               | Yes     |         |                     |   No    |                |
+| Service      | Yes        | Yes[^1]  | Yes[^1][^2]       | Yes[^3] | Yes     | Yes                 |   No    |      Yes       |
+| Skipper      | Yes        | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |                |
+| Traefik      |            | Yes[^1]  |                   | Yes     | Yes     | Yes                 |   No    |                |
```

Comment:
🤔 Since this annotation applies only on `Service`, I'm not sure if it's relevant to add it in this table overview.

Reply:
If we follow this path of adding all annotations to this table, even those which are specific to one source like `access` or `ingress-hostname-source`, that would impact the readability ? or did I miss something ?

Reply:
Yes, it makes sense to me. Please remove those two. 
It will keep this table focused on cross-source annotations.

Reply:
Updated